test/e2e/flowexporter_test.go

```diff
@@ -118,19 +169,28 @@ func TestFlowExporter(t *testing.T) {
 
 		if strings.Contains(record, podAIP) && strings.Contains(record, podBIP) {
 			dataRecordsIntraNode = dataRecordsIntraNode + 1
-			// Check if records have both Pod name and Pod namespace or not
+			// Check if records have both Pod name and Pod namespace or not.
 			if !strings.Contains(record, hex.EncodeToString([]byte("perftest-a"))) {
-				t.Fatalf("Records with podAIP does not have pod name")
+				t.Fatalf("Records with PodAIP does not have Pod name")
 			}
 			if !strings.Contains(record, hex.EncodeToString([]byte("perftest-b"))) {
-				t.Fatalf("Records with podBIP does not have pod name")
+				t.Fatalf("Records with PodBIP does not have Pod name")
 			}
 			if !strings.Contains(record, hex.EncodeToString([]byte(testNamespace))) {
-				t.Fatalf("Records with podAIP and podBIP does not have pod namespace")
+				t.Fatalf("Records with PodAIP and PodBIP does not have Pod Namespace")
+			}
+			// Check if records have both ingress and egress network policies.
+			if !strings.Contains(record, "octetDeltaCount: 0") {
```

Comment:
that's a weird condition, could you add a comment?

Reply:
Added.